pyanaconda/ui/gui/xkl_wrapper.py

```diff
@@ -35,13 +35,18 @@
 
 import threading
 import gettext
+import locale as locale_mod
 from collections import namedtuple
 
 from pyanaconda.core import util
 from pyanaconda.core.configuration.anaconda import conf
 from pyanaconda.core.constants import DEFAULT_KEYBOARD
-from pyanaconda.keyboard import join_layout_variant, parse_layout_variant, KeyboardConfigError, InvalidLayoutVariantSpec
+from pyanaconda.keyboard import (
```

Comment:
Please, don't use parentheses for import. Just end the line with `\` and continue on the next one.